src/Fields.php

```diff
@@ -1010,6 +1010,9 @@ protected function wf_crm_get_fields($var = 'fields') {
           $fields[$id]['extra']['description_above'] = (int) empty($custom_field['help_pre']);
           $fields[$id]['has_help'] = TRUE;
         }
+        if ($custom_field['serialize'] || FALSE) {
```

Comment:
@MegaphoneJon can we get rid of extra `|| FALSE` condition?

Reply:
Are we certain that the `serialize` key will always be present?  This is meant to prevent an e-notice if it's not (comparable to using `isset()`.

Reply:
> Are we certain that the serialize key will always be present?

I do get `serialize` for every custom field `get` using API explorer (and on the webform). But i do agree a safe check is required. 

> This is meant to prevent an e-notice if it's not (comparable to using isset().

Ah, is it? It seems `$custom_field['serialize']` will still be checked and trigger e-notice if the key isn't present? 

I tried to unset the key just before this line to test and confirmed that the notice is displayed even with the extra `|| FALSE`. Can you pls clarify? or can we replace it with simple !empty() check since we only need to enter this when `serialize` is present and is equal to 1. Agree @MegaphoneJon ?

Reply:
Right, the addition of `|| FALSE` doesn't add anything to an if statement. It doesn't guard against e-notices and it doesn't affect the logic; it can be removed.
If there is a chance of the key not existing, you can change the line to:
```suggestion
        if (!empty($custom_field['serialize'])) {
```